src/core/importType.js

```diff
@@ -43,7 +43,7 @@ export function isExternalModuleMain(name, settings, path) {
   return externalModuleMainRegExp.test(name) && isExternalPath(path, name, settings)
 }
 
-const scopedRegExp = /^@[^/]+\/[^/]+/
+const scopedRegExp = /^@[^/]+\/?[^/]+/
```

Comment:
hmm, this also seems to allow `@foo/` - can you add an explicit test for that?

Reply:
When I tested, the '@foo/` case is recognized as `external`.
Is it correct, or must I change that?

Reply:
@ljharb I re-up this question

Reply:
I think that because it's got the `/`, it's a folder in node_modules, so external seems right?

Reply:
I have the same opinion on this point.
So it seems ok since there's a test for `@cycle/`.